lib/components/Notification/banner/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useRef, useState } from "react";
+import clsx from "clsx";
+import { TYPE } from "@utils/notification-utils";
+import { Notification } from "../base";
+import type { NotificationProps } from "../base";
+
+export interface NotificationBannerProps
+    extends Omit<
+        NotificationProps,
+        | "isBanner"
+        | "status"
+        | "hasCloseButton"
+        | "onSwipeDown"
+        | "onSwipeLeft"
+        | "onSwipeRight"
+        | "onSwipeUp"
+    > {
+    autohideTimeout?: number;
```

Comment:
we need to initialise with default time no ?

Reply:
i saw where u initialised the timeout

Reply:
it's also not a bad approach. instead of adding or statement we can have `4000` as default value of `autohideTimeout`

Reply:
no, we don't. the default time should be set for the banner based on the needs of business: that's why we pass it as `autohideTimeout={autohideTimeout ?? 4000}`.
Since no default time is set, a single banner component will not disappear and it will allow QA to test the component properly.

Reply:
if that's the case, we can leave it like this and update once they have decided that we really need a fixed time for it.